packages/react-charts/src/components/ChartCursorTooltip/ChartCursorTooltip.tsx

```diff
@@ -12,8 +12,9 @@ import {
 import { VictoryTooltip } from 'victory-tooltip';
 import { ChartLabel } from '../ChartLabel';
 import { ChartThemeDefinition } from '../ChartTheme';
-import { ChartTooltip, ChartTooltipProps } from '../ChartTooltip';
-import { getTheme, getCursorTooltipCenterOffset, getCursorTooltipPoniterOrientation } from '../ChartUtils';
+import { ChartTooltip, ChartTooltipProps } from '../ChartTooltip/ChartTooltip';
+import { getTheme } from '../ChartUtils/chart-theme';
+import { getCursorTooltipCenterOffset, getCursorTooltipPoniterOrientation } from '../ChartUtils/chart-tooltip';
```

Comment:
Can you please explain these changes? I'm afraid these types of things may be reverted without some comments as to why the module index cannot be used.

Reply:
I'll revert charts changes, I just fixed them as a best-effort to adhere to good coding practices. These are the errors it fixed if we were to bundle charts as UMD:
```sh
(!) Circular dependency: dist/esm/components/ChartUtils/index.js ->
  dist/esm/components/ChartUtils/chart-container.js ->
  dist/esm/components/ChartCursorTooltip/index.js ->
  dist/esm/components/ChartCursorTooltip/ChartCursorTooltip.js ->
  dist/esm/components/ChartTooltip/index.js ->
  dist/esm/components/ChartTooltip/ChartTooltip.js ->
  dist/esm/components/ChartUtils/index.js
(!) Circular dependency: dist/esm/components/ChartUtils/index.js ->
  dist/esm/components/ChartUtils/chart-container.js ->
  dist/esm/components/ChartCursorTooltip/index.js ->
  dist/esm/components/ChartCursorTooltip/ChartCursorTooltip.js ->
  dist/esm/components/ChartUtils/index.js
```